packages/admin-stories/src/docs/components/Table/FilterBar/FilterBar.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+
+<Meta title="Docs/Components/Table/Filterbar/FilterBar" />
+
+# Filterbar
+
+The Filterbar Component helps you create a horizontal orientated list of PopoverFields which can include all kind of sorts of Final Form Fields.
+The Intention of the Filterbar was to act as a Filter of a Table, which is also the main usecase for this Component.
+**The Filterbar only works inside of a Final Form.**
+
+The Filterbar consists of four different components:
+
+-   FilterBar
+-   FilterBarPopoverFilter
+-   FilterBarMoreFilters
+-   FilterBarActiveFilterBadge
+
+## Table Example
+
+<Canvas>
+    <Story id="stories-components-table-filterbar-filterbar-default--default" />
+</Canvas>
+
+## Advanced Informations
+
+Every FilterBarPopoverFilter Component contains its own FinalForm. So the Filterbar is kinda like a collection of final forms, which pass their values into a wrapping form.
```

Comment:
This is an implementation detail. I don't think potential users need to know this